solr/core/src/test/org/apache/solr/search/FuzzySearchTest.java

```diff
@@ -51,6 +52,7 @@ public void setupCollection() throws Exception {
   }
 
   @Test
+  @Ignore //  no longer too complex in latest Lucene ...
```

Comment:
Do we have an idea of q that _would_ be too complex? If not, let's delete the whole test instead of leaving it around and ignored.

Reply:
Nothing I found back when I tried a few things to still tickle it. Doesn't mean there is not something good, but would take more than some attempted guesses from me to hit it.